evm/vm/code_stream.py

```diff
@@ -10,13 +10,15 @@
 
 class CodeStream(object):
     stream = None
+    deepest = None
 
     logger = logging.getLogger('evm.vm.CodeStream')
 
     def __init__(self, code_bytes):
         validate_is_bytes(code_bytes)
         self.stream = io.BytesIO(code_bytes)
-        self._validity_cache = {}
+        self._validity_cache = set(range(0))
```

Comment:
This should probably be changed to just `set()` as I think the `range(0)` part is not necessary.  And can we now rename this to `invalid_positions` since it's no longer really a cache.